code/modules/admin/verbs/randomverbs.dm

```diff
@@ -1598,3 +1557,179 @@ Traitors and the like can also be revived with the previous role mostly intact.
 			return
 		else
 			usr.forceMove(T)
+
+/// The default command report announcement sound.
+#define DEFAULT_ANNOUNCEMENT_SOUND "default_announcement"
+
+/// Preset central command names to chose from for centcom reports.
+#define CENTCOM_PRESET "Central Command"
+#define SYNDICATE_PRESET "The Syndicate"
+#define WIZARD_PRESET "The Wizard Federation"
+#define CUSTOM_PRESET "Custom Command Name"
+
+/// Verb to change the global command name.
+/client/proc/cmd_change_command_name()
+	set category = "Admin.Events"
```

Comment:
i didnt mean move it to this file i meant the set category part should be "Admin.Round Interaction" like it used to be